includes/admin/class-wc-payments-admin.php

```diff
@@ -836,4 +837,25 @@ public function maybe_redirect_to_onboarding() {
 
 		$this->account->redirect_to_onboarding_page();
 	}
+
+	/**
+	 * Add woopay as a payment method to the edit order on admin.
+	 *
+	 * @param int $order_id order_id.
+	 */
+	public function show_woopay_payment_method_name_admin( $order_id ) {
+		$order = new WC_Order( $order_id );
+		if ( ! $order->get_meta( 'is_woopay' ) ) {
+			return;
+		}
+		?>
+		<div class="wc-payment-gateway-method-name-woopay-wrapper">
+			<?php echo esc_html_e( 'Paid with ', 'woocommerce-payments' ); ?>
```

Comment:
For here and the `'Card ending in '` string below, can we move the trailing whitespace outside of the internationalized string? I think generally it's best for strings that we expect to be translated to not have trailing whitespace that we rely on, since that might not get included in translation.

Reply:
Fixed with 6a1f0bb66b58fa48d70cf228c4221554baed3a51